pom.xml

```diff
@@ -137,6 +137,12 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>org.apache.maven.plugin-tools</groupId>
+        <artifactId>maven-plugin-annotations</artifactId>
+        <version>${pluginPluginVersion}</version>
+        <scope>provided</scope>
```

Comment:
Please do not use `<scope>` under dependencyManagement. Having it there is not reliable and thus redundant.

Reply:
I can remove it but I think without it, some dependency can then force  it into the scope compiled. As it is, even no scope is given at other places, it should be in `provided` in the final classpath.
Am I wrong?

Reply:
SO seems to support my approach: https://stackoverflow.com/questions/15221299/dependencymanagement-and-scope

Reply:
I frankly do not know what happens if `maven-plugin-annotations` accidentally get into `compile` scope. Having `<scope>provided</scope>` only in the dependencies section worked so far. So let's keep the old strategy and remove the scopes from the dependencyManagement.